packages/EasyCodingStandard/packages/FixerRunner/src/Application/FixerFileProcessor.php

```diff
@@ -87,7 +87,7 @@ public function addFixer(FixerInterface $fixer): void
     }
 
     /**
-     * @return FixerInterface[]
+     * @return mixed[] Fixers that implements FixerInterface
```

Comment:
Why? This makes no sense

Imagine this

```diff
 /**
- * @var FixerInterface[]
+ * @var mixed[] Array with FixerInterface elements
  */
```

Crazy

Reply:
Because other methods return are broke, because isn't a `FixerInterface`

Reply:
What is it then?

This method should only reutnr `FixerInterface`